### PR TITLE
OP-543 message cleanup and consolidation

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-543

### CODE REVIEW

File: src/main/java/org/isf/visits/gui/InsertVisit.java [hunk 1]
```diff
@@ -448,7 +448,7 @@ public void actionPerformed(ActionEvent e) {
 	private JPanel getPanelChoosePatient() {
 		JPanel choosePatientPanel = new JPanel();
 		choosePatientPanel.setLayout(new FlowLayout(FlowLayout.CENTER));
-		choosePatientPanel.setBorder(BorderFactory.createTitledBorder(MessageBundle.getMessage("angal.common.pleaseselectapatient"))); //$NON-NLS-1$
+		choosePatientPanel.setBorder(BorderFactory.createTitledBorder(MessageBundle.getMessage("angal.visit.pleaseselectapatient.border")));
```
Comment (mwithi) [inline src/main/java/org/isf/visits/gui/InsertVisit.java:451]: The "good" to have duplicated keys is to have all labels concerning the same package under the same root key and of course to provide specific translations for some of them (and also the original intention though); the "evil" si redundancy of texts and translations. 
Balancing between the two strategy is something we should document somewhere for devs, just a note!

**Review by mwithi — APPROVED** [review]
LGTM, only some comments

File: bundle/language_en.properties [hunk 2]
```diff
@@ -1038,148 +1034,42 @@ angal.ward.theemailmustbevalid.msg
 angal.ward.theselectedwardhaspatients.fmt.msg                                    = The selected ward has {0} patients.
 angal.ward.pleasecheckinadmissionpatients.msg                                    = Please check in admission / patients.
 angal.ward.maternity.txt                                                         = Maternity
-
+angal.ward.cannotdeletematernityward.msg                                         = Cannot delete maternity ward.
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply bundle/language_en.properties:1037]: Done; thanks.

File: bundle/language_en.properties [hunk 3]
```diff
@@ -1564,95 +1302,39 @@ angal.medicalstockward.rectify.pleaseinsertavalidvalue
...
-angal.medicalstockward.weightto                                                  = To
 angal.medicalstockwardedit.age                                                   = Age
 angal.medicalstockwardedit.changepatient                                         = Change Patient
 angal.medicalstockwardedit.changethepatientassociatedwiththismovement            = Change the Patient associated with this movement
 angal.medicalstockwardedit.drug                                                  = Drug
-angal.medicalstockwardedit.insertquantitypiecesormls                             = Insert quantity (pieces/mls):
 angal.medicalstockwardedit.insertvalidage                                        = Insert a valid age
-angal.medicalstockwardedit.instock                                               = In stock
 angal.medicalstockwardedit.internaluse                                           = Internal use
-angal.medicalstockwardedit.invalidproduct                                        = Invalid Product
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply bundle/language_en.properties:1593]: This key doesn't exist anywhere in Core or GUI repos.

File: src/main/java/org/isf/visits/gui/InsertVisit.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/visits/gui/InsertVisit.java:451]: I am going to make another pass through the keys and handle `title` and `border` as mentioned in the other PR.

Comment (mwithi) [reply src/main/java/org/isf/visits/gui/InsertVisit.java:451]: ok

**Pull request merged**